src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,55 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.ProductDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("/products")
+public class ProductController {
+
+
+    @GetMapping
+    public ResponseEntity<List<ProductDto>> getAllProducts() {
+        List<ProductDto> exampleList = new ArrayList<>();
+
+        for (long i = 1; i < 6; i++) {
+            exampleList.add(new ProductDto(
+                    i, "Product " + i, "Description " + i, new BigDecimal(10*i), 2));
+        }
+        return ResponseEntity.ok(exampleList);
+    }
+
+
+
+    @GetMapping("/{id}")
+    public ResponseEntity<ProductDto> getProduct(@PathVariable final Long id) {
+        return ResponseEntity.ok(new ProductDto(
+                1L, "Product "+id, "Example description", new BigDecimal(150), 3));
+    }
+
+
+    @PostMapping()
+    public ResponseEntity<ProductDto> addNewProduct(@RequestBody final ProductDto product) {
+        return ResponseEntity.ok(product);
```

Comment:
I think that this method should map ProductDto to Product and put it into repository.
Is should return "void"